### PR TITLE
merge and modification of the rez spell pr.

### PR DESCRIPTION
**Pull request by RekkasGit**

### CODE REVIEW

File: E3Next/Settings/CharacterSettings.cs [hunk 1]
```diff
@@ -337,7 +342,17 @@ private void LoadData()
 
 
             LoadKeyData("Heals", "Tank", ParsedData, HealTankTargets);
+            for (Int32 i = 0; i < HealTankTargets.Count; i++)
+            {
+                HealTankTargets[i] = e3util.FirstCharToUpper(HealTankTargets[i]);
```
Comment (jonnyrocket0) [inline E3Next/Settings/CharacterSettings.cs:347]: Why not just use `TextInfo.ToTitleCase()`?

File: E3Next/Settings/CharacterSettings.cs [hunk 2]
```diff
@@ -337,7 +342,17 @@ private void LoadData()
...
+            }
+
             LoadKeyData("Heals", "Important Bot", ParsedData, HealImportantBotTargets);
+            //upper case first letter on all important bots, netbots bug that doesn't like lower case.
+            for (Int32 i = 0; i < HealImportantBotTargets.Count; i++)
+            {
+                HealImportantBotTargets[i] = e3util.FirstCharToUpper(HealImportantBotTargets[i]);
```
Comment (jonnyrocket0) [inline E3Next/Settings/CharacterSettings.cs:354]: Same

File: E3Next/Settings/CharacterSettings.cs [hunk 3]
```diff
@@ -135,6 +135,10 @@ public class CharacterSettings : BaseSettings, IBaseSettings
         public List<Spell> HealPets = new List<Spell>();
         public List<Spell> HealOverTime = new List<Spell>();
         public List<string> HealPetOwners = new List<string>();
+        //rez spells
+        public List<string> Rez_RezSpells = new List<string>();
+        public bool Rez_AutoRez = false;
```
Comment (jonnyrocket0) [inline E3Next/Settings/CharacterSettings.cs:140]: The names feel redundant

**Comment by jonnyrocket0** [comment]
This is a breaking change, so we need to be sure to communicate that clearly.

File: E3Next/Settings/CharacterSettings.cs (hunk 1 above)
Comment (RekkasGit) [reply E3Next/Settings/CharacterSettings.cs:347]: https://itecnote.com/tecnote/c-make-first-letter-of-a-string-upper-case-with-maximum-performance/

File: E3Next/Settings/CharacterSettings.cs (hunk 2 above)
Comment (RekkasGit) [reply E3Next/Settings/CharacterSettings.cs:354]: https://itecnote.com/tecnote/c-make-first-letter-of-a-string-upper-case-with-maximum-performance/

File: E3Next/Settings/CharacterSettings.cs (hunk 3 above)
Comment (RekkasGit) [reply E3Next/Settings/CharacterSettings.cs]: fixed

File: E3Next/Settings/CharacterSettings.cs (hunk 2 above)
Comment (jonnyrocket0) [reply E3Next/Settings/CharacterSettings.cs:350]: TIL

**Review by jonnyrocket0 — APPROVED** [review]

**Pull request merged**